apps/mobile/src/components/ChatDetailScreen/components/MessagesList.tsx

```diff
@@ -1,11 +1,21 @@
 import {FlashList} from '@shopify/flash-list'
 import {useMolecule} from 'bunshi/dist/react'
-import {useAtomValue, type Atom} from 'jotai'
+import {useAtomValue, useSetAtom, type Atom} from 'jotai'
 import React from 'react'
+import {StyleSheet} from 'react-native'
 import atomKeyExtractor from '../../../utils/atomUtils/atomKeyExtractor'
 import {chatMolecule} from '../atoms'
 import MessageItem, {type MessagesListItem} from './MessageItem'
 
+const LIST_ITEM_VISIBILITY_PERCENTAGE_THRESHOLD = 30
+const QUICK_ACTION_BANNER_HEIGHT_WITH_PADDING = 80
```

Comment:
Use the same constant for setting height of quick action banner so if we decide to change height of quick action banner we don't break UX